update salvatore 2020 (#104)
shedding-hub/shedding-hub: data/salvatore2020epidemiological/salvatore2020epidemiological.md

# Extraction for Salvatore et al. (2020)

> This study was conducted in Utah and Wisconsin between 23 March and 13 May 2020, with testing data collected during a prospective household transmission investigation of outpatient and mild coronavirus disease 2019 cases.

Reviewer: Please add back the link to the paper.
```suggestion
[Salvatore et al. (2020)](https://pmc.ncbi.nlm.nih.gov/articles/PMC7543310/) was conducted in Utah and Wisconsin between 23 March and 13 May 2020, with testing data collected during a prospective household transmission investigation of outpatient and mild coronavirus disease 2019 cases.
```

Author: Added.

Results were considered positive if signals were detected (Ct < 40) for the RP, N1, and N2 genes. Results were classified as “not detected” if RP was detected but no signal was observed (Ct ≥ 40) from either N1 or N2. Results were classified as inconclusive if RP was detected (Ct < 40) and either N1 or N2 was detected (but not both). Results were classified as invalid if no RP was detected in the sample. Any specimens for which results were inconclusive were retested; specimens which produced inconclusive results after retesting were excluded from the analysis (n = 17 specimens from 6 participants). Ct values for amplification of both viral targets (N1 and N2 probes).

> The author of the original paper stated: "For this analysis, we focused on values of the N1 probe." Thus, we think the viral load in data is from N1 probe.

Reviewer: ```suggestion
The author of the original paper stated: "For this analysis, we focused on values of the N1 probe." Thus, we believe the CT values in the data are from the N1 probe.
```

Author: Fixed

from shedding_hub import folded_str
```

```python
#load the data;
df = pd.read_excel('CombinedDataset.xlsx', sheet_name='Viral_Load')
df = df[df['StudyNum']==17]
```

```python
#Drop unnecessary columns
df = df.loc[:,'Day':'Sex'].join(df.loc[:,['Ctvalue','LOD','PatientID']])
```

```python
#Modify the values to fit the schema
df['Sex'] = df['Sex'].replace({'M':'male','F':'female'})
df['Ctvalue'] = df['Ctvalue'].astype('object')
df['Age'] = df['Age'].astype('object')
df['Day'] = df['Day'].astype('object')
df.loc[df['Ctvalue']==1,'Ctvalue'] = 'negative'
#Rename column
df.rename(columns={'PatientID': 'id'}, inplace=True)
```

```python
#Output Yaml file
#Enter the yaml writing stage
participant_list = [dict(attributes=dict(age=df.loc[df.loc[df["id"]==i].index[0],'Age'],
                            sex=df.loc[df.loc[df["id"]==i].index[0],'Sex']),

>                             measurements=[dict(analyte='N1_and_N2_probes',time=j,

Reviewer: As we state this is only N1, could you rename this?

Author: Renamed

                                            value=df.loc[(df['Day'] == j) & (df['id'] == i),"Ctvalue"].item()) for j in np.unique(df.loc[df['id'] == i,'Day'])]) for i in np.unique(df['id'])]

```

```python
phillip2020 = dict(title="Epidemiological Correlates of Polymerase Chain Reaction Cycle Threshold Values in the Detection of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)",
               doi="10.1093/cid/ciaa1469",
               description=folded_str('This study was conducted in Utah and Wisconsin between 23 March and 13 May 2020, with testing data collected during a prospective household transmission investigation of outpatient and mild coronavirus disease 2019 cases.\n'),

>                analytes=dict(N1_and_N2_probes=dict(description=folded_str("SARS-CoV-2 RNA genome copy concentration calculated from evaluation of both N1 and N2 probe.\n Results were considered positive if signals were detected (Ct < 40) for the RP, N1, and N2 genes. Results were classified as 'not detected' if RP was detected but no signal was observed (Ct >= 40) from either N1 or N2. Results were classified as inconclusive if RP was detected (Ct < 40) and either N1 or N2 was detected (but not both). Results were classified as invalid if no RP was detected in the sample. Any specimens for which results were inconclusive were retested; specimens which produced inconclusive results after retesting were excluded from the analysis (n = 17 specimens from 6 participants). Ct values for amplification of both viral targets (N1 and N2 probes).\n"),

Reviewer: Same here. Rename the `analyte`.

Author: Fixed

                                                    specimen="nasopharyngeal_swab",
                                                    biomarker="SARS-CoV-2",

>                                                     gene_target="N1 probe",

Reviewer: Delete  " probe" just using "N1".

Author: Deleted
